pkg/controllers/audit/webhook.go

```diff
@@ -26,10 +27,22 @@ func NewController(logger *zap.SugaredLogger, client hec.HEC) *Controller {
 
 // AuditEvent handles an audit event
 func (c *Controller) AuditEvent(response http.ResponseWriter, request *http.Request) {
-	c.logger.Infow("received audit event", "request", request.Body)
-
-	event := hec.NewEvent("event one")
-	event.SetTime(time.Now())
+	Body, _ := ioutil.ReadAll(request.Body)
```

Comment:
It confuses me that you started this variable with a capital letter. I would expect that only "public" variables should start with a capital letter while function scope variables should start with small letters. However, I cannot find a word about using capital letters in function scope variables in any Golang style guide and golang linters and IDEs seem fine with this. Mh.

Reply:
Makes sense anyway - changed as you suggested.

Reply:
Thanks. I maybe I find evidence one day. :laughing: